src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -197,6 +197,11 @@ if (NGRAPH_CPU_ENABLE)
          message(WARNING "The build toolset doesn't support OpenMP. This will impact performance and lead to slowdowns.")
     endif()
 
+    if (MSVS)
```

Comment:
What is the difference between WIN32 and MSVS? Can we just use one or different meanings?

Reply:
WIN32 works for both LLVM and VC. These compile options only works for VC.

Reply:
MSVS is a variable that I created
See https://github.com/NervanaSystems/ngraph/blob/master/CMakeLists.txt#L67
It means Visual Studio is used. So it is different from WIN32
Windows build may use a different generator like Ninja if we add support in the future.